src/test/groovy/com/stormpath/tck/authentication/BearerIT.groovy

```diff
@@ -0,0 +1,23 @@
+package com.stormpath.tck.authentication
+
+import com.stormpath.tck.AbstractIT
+import org.testng.annotations.Test
+
+import static com.jayway.restassured.RestAssured.given
+import static com.stormpath.tck.util.FrameworkConstants.MeRoute
+
+class BearerIT extends AbstractIT {
+
+    @Test(groups=["v100", "json", "html"])
+    public void refreshTokenAsAccessBearerTokenFails() throws Exception {
```

Comment:
Isn't this covered here? 

https://github.com/stormpath/stormpath-framework-tck/blob/adc6b64a1bfbee0b71b45772dfc695678df42fb1/src/test/groovy/com/stormpath/tck/authentication/CookieIT.groovy#L149